test/workers/job_create_worker_test.rb

```diff
@@ -0,0 +1,89 @@
+require 'test_helper'
+
+class JobCreateWorkerTest < ActiveSupport::TestCase
+
+  before {
+    ENV['WORKER_LIB'] = 'inline'
+    ActiveJob::Base.queue_adapter = :inline
+  }
+
+  after {
+    ENV['WORKER_LIB'] = 'test'
+    ActiveJob::Base.queue_adapter = :test
+  }
+
+
+  let(:user) { User.create!(email: 'test@prx.org', password: 'foobarpassword') }
+
+  let(:application) do
+    Doorkeeper::Application.create(
+      name: 'test',
+      owner: user,
+      redirect_uri: 'urn:ietf:wg:oauth:2.0:oob'
+    )
+  end
+
+  let(:msg) do
+    {
+      client_id: application.uid,
+      job_type: 'test',
+      priority: 1,
+      retry_max: 10,
+      retry_delay: 300,
+      tasks: [
+        {
+          task_type: 'echo',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.